paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -306,6 +309,7 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
     }
 
     fun handlePrimaryButtonClick(screenState: USBankAccountFormScreenState) {
+        handlePrimaryButtonStateChanged(PrimaryButton.State.StartProcessing)
```

Comment:
Is this required in all cases or only in `BillingDetailsCollection` (as the other ones don’t do a network call, if I remember correctly)?